### PR TITLE
[strings - categories] Updated (and added missing) Turkish translations

### PR DESCRIPTION
**Pull request by MetehanOzyurek**

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -836,7 +836,7 @@ es:3Restaurante|3café
 eu:3Jatetxea|3Kafetegia
 sv:3Restaurang|3café
 th:3ร้านอาหาร
-tr:3Restoran|3kafeterya
+tr:2Restoran|3kafeterya|2lokanta|2kafe
```
Comment (biodranik) [inline data/categories.txt:839]: Why you decided to use 2? It is usually a very low number, considering that restaurants are already searched by "Where to eat" shortcut.

File: data/categories.txt [hunk 2]
```diff
@@ -904,7 +904,7 @@ es:3Panadería
 eu:3Okindegia
 sv:3Bageri
 th:3เบเกอรี่
-tr:3Fırın
+tr:2Fırın
```
Comment (biodranik) [inline data/categories.txt:907]: The same. Why 2?

File: data/categories.txt [hunk 3]
```diff
@@ -1703,7 +1706,7 @@ es:3Supermercado
 eu:3Supermerkatua
 sv:3Stormarknad|dagligvaruhandel
 th:3ซูเปอร์มาร์เก็ต
-tr:3Süpermarket
+tr:2Süpermarket|3market
```
Comment (biodranik) [inline data/categories.txt:1709]: Why 2?

File: data/categories.txt [hunk 4]
```diff
@@ -8469,7 +8472,7 @@ es:Zona de caravanas|zona de camping
 eu:Karabana gunea|kanpalekua
 sv:4Husvagnsplats
 th:4ขบวนนักเดินทาง
-tr:4Karavan alanı
+tr:4Karavan alanı|karavan parkı
```
Comment (biodranik) [inline data/categories.txt:8475]: karavan?

File: data/categories.txt (hunk 2 above)
Comment (MetehanOzyurek) [reply data/categories.txt:907]: I thought it would better
I can change if it's problem

**Comment by MetehanOzyurek** [comment]
@biodranik is it OK or should I change 2 to 3?

**Comment by biodranik** [comment]
1. We prefer to avoid 2, because the chance is very high that the user types something else.
2. If you have two words in categories without a number, then the user should type these two words exactly for the search to work. Compare it with typing only one word exactly.

**Comment by MetehanOzyurek** [comment]
> 1. We prefer to avoid 2, because the chance is very high that the user types something else.
> 2. If you have two words in categories without a number, then the user should type these two words exactly for the search to work. Compare it with typing only one word exactly.

OK got it. So I'll edit asap.

**Review by biodranik — APPROVED** [review]

**Pull request merged**